aws/provider.go

```diff
@@ -255,6 +255,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_network_interface":                         dataSourceAwsNetworkInterface(),
 			"aws_network_interfaces":                        dataSourceAwsNetworkInterfaces(),
 			"aws_organizations_organization":                dataSourceAwsOrganizationsOrganization(),
+			"aws_organizations_organizational_unit":         dataSourceAwsOrganizationsOrganizationalUnit(),
```

Comment:
This data source as written appears to be what we typically consider a "plural" data source, one that returns multiple infrastructure components from a single lookup. We prefer to suffix these with a `s`, e.g. `aws_organizations_organizational_units`, to help operators know that this can/will return multiple results.

Sorry for the hassle but can you please do the following here:

- Update the file naming to be plural, e.g. `aws/data_source_aws_organizations_organization_units.go`, `website/docs/d/organizations_organization_units.html.markdown`
- Update the function naming to be plural, e.g. `dataSourceAwsOrganizationsOrganizationalUnits()`, `dataSourceAwsOrganizationsOrganizationalUnitsRead()`
- Update the data source naming

```suggestion
			"aws_organizations_organizational_units":        dataSourceAwsOrganizationsOrganizationalUnits(),
```

Reply:
No problem!